docs/start/authentication/graphql-storefront.mdx

```diff
@@ -66,17 +66,227 @@ content-type: application/json
+"pass": "password"
+}
+```
+</Tab>
+<Tab>
+
+```json filename="Example login mutation: Create a customer access token" showLineNumbers copy
+{
+    "data": {
+        "login": {
+            "result": "success",
+            "customer": {
+                "entityId": 21,
+                "email": "user@email.com"
+            },
+            "customerAccessToken": {
+                "value": "53ea9099d5484cdedddbf8f59a73bc284afefd7943c4dbe578db52435e0c0de7_1721406629",
+                "expiresAt": "2024-07-19T16:30:29Z"
+            }
+        }
+    }
+}
+```
+</Tab>
+</Tabs>
+
+###### Login with a JWT
+
+The second option is to use the `loginWithCustomerLoginJwt` mutation to pass your customer JSON web token instead of the user's email and password. In this scenario, you will generate a JWT that contains key information in the payload for the login to be successful. You will use this JWT in the login mutation to receive a customer access token.
+
+<Callout type="info">
+For information on creating a JWT, see the [Customer Login API](/docs/start/authentication/customer-login) article for instructions.
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```graphql filename="Example JWT login mutation: Create a customer access token" showLineNumbers copy
+# Creates a customer access token
+POST https://{{storeDomain}}/graphql
+Authorization: Bearer {Storefront token}
+accept: application/json
+content-type: application/json
+
+mutation Login($jwt: String!) {
+ loginWithCustomerLoginJwt(jwt: $jwt) {
+   customer {
+     entityId
+     email
+   }
+   customerAccessToken {
+     value
+     expiresAt
+   }
+ }
+}
+```
+
+```json filename="GraphQL variables" showLineNumbers copy 
+{"jwt": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiI3MjM3Mzc2Y3JiNnJrcTFhY2h3MmdrdTQ2M2FjeGk4IiwiaWF0IjoxNzIwODE0NjgyLCJqdGkiOiJhMDA1OGQ0MC0zNmExLTAxM2ItOTBmNC03ZThiMTE1MDA4YTIiLCJvcGVyYXRpb24iOiJjdXN0b21lcl9sb2dpbiIsInN0b3JlX2hhc2giOiJ3MGFzdGVlOWpvIiwiY3VzdG9tZXJfaWQiOjIyfQ.bfiEpk-IDCqQ4DqpErVCEUUQDjs-vtWxHsEufkUFqMs"
+}
+```
+</Tab>
+<Tab>
+
+```json filename="Example JWT login mutation: Create a customer access token" showLineNumbers copy
+{
+    "data": {
+        "loginWithCustomerLoginJwt": {
+            "customer": {
+                "entityId": 22,
+                "email": "user@email.com"
+            },
+            "customerAccessToken": {
+                "value": "ee973d6566a7c691af4602f344d866ecaa45bdcbdf2075d29d5e70fd0716b7c5_1721657159",
+                "expiresAt": "2024-07-22T14:05:59Z"
+            }
+        }
+    }
+}
+```
+</Tab>
+</Tabs>
+
+
+##### Using a Customer Access Token
+
+You can assign the customer access token obtained using one of the mutations described above to the `x-Bc-Customer-Access-Token` header for future queries. You can use this token for a specific shopper session access until it is invalidated. The token is invalidated when you use the logout mutation with the `x-Bc-Customer-Access-Token` header.
```

Comment:
```suggestion
You can assign the customer access token obtained using one of the mutations described above to the `X-Bc-Customer-Access-Token` header for future queries. You can use this token for a specific shopper session access until it is invalidated. The token is invalidated when you use the logout mutation with the `X-Bc-Customer-Access-Token` header.
```
For headers it doesn't make a difference if letter is uppercase or lowercase, but I think usually in text we put this first `X` also big.